env-example

```diff
@@ -78,7 +78,6 @@ PHP_FPM_INSTALL_EXIF=false
 PHP_FPM_INSTALL_AEROSPIKE=false
 PHP_FPM_INSTALL_MYSQLI=false
 PHP_FPM_INSTALL_PGSQL=false
-PHP_FPM_INSTALL_POSTGRES=false
```

Comment:
Duplicate?

Reply:
PHP_FPM_INSTALL_PGSQL=false
same
PHP_FPM_INSTALL_POSTGRES=false

Reply:
`PHP_FPM_INSTALL_PGSQL` for pg client.
`PHP_FPM_INSTALL_POSTGRES ` for pg server.

Reply:
## see: https://github.com/laradock/laradock/blob/master/php-fpm/Dockerfile-71

54:
```
#####################################
# pgsql
#####################################

ARG INSTALL_PGSQL=false
RUN if [ ${INSTALL_PGSQL} = true ]; then \
    # Install the pgsql extension
    apt-get update -yqq && \
    docker-php-ext-install pgsql \
;fi
```


241:

```
#####################################
# postgres Modifications:
#####################################

ARG INSTALL_POSTGRES=false
RUN if [ ${INSTALL_POSTGRES} = true ]; then \
    docker-php-ext-install pgsql \
;fi
```

Reply:
```
#####################################
# pgsql
##################################### 
......
docker-php-ext-install pgsql 
```
for pg client?

```
#####################################
# postgres Modifications:
#####################################
......
 docker-php-ext-install pgsql 
```
 for pg server？   why？ It is  same！